eng/scripts/build.ps1

```diff
@@ -1,12 +1,20 @@
 #Requires -Version 7.0
-param($filter, [switch]$vet, [switch]$generate, [switch]$skipBuild, $parallel = 5)
+param($filter, [switch]$clean, [switch]$vet, [switch]$generate, [switch]$skipBuild)
 
+$startingDirectory = Get-Location
+$root = Resolve-Path ($PSScriptRoot + "/../..")
+Set-Location $root
 $sdks = @{};
 
 foreach ($sdk in (./eng/scripts/get_module_dirs.ps1 -serviceDir 'sdk/...')) {
```

Comment:
I don't think it matters much right now, but depending on whether get_module_dirs is also called outside of powershell, it may be a better pattern (for readability, maintenance, etc.) to import functions from this as a module instead of calling out to the scripts.

Reply:
I'm not opposed to this, i just didn't know you could do that.